runtime/src/main/java/org/corfudb/util/serializer/DynamicProtobufSerializer.java

```diff
@@ -284,5 +291,26 @@ public void serialize(Object o, ByteBuf b) {
             throw new SerializerException(ie);
         }
     }
+
+    public String toString() {
+        StringBuilder stringBuilder = new StringBuilder();
```

Comment:
it will be very hard to parse these kinds of messages in log aggregation platforms like ELK.
Can you use one string for that?

Reply:
hi @xnull sorry I missed this comment before merge. I would like to address it. Could you please share details on what is the ELK format for strings?

Reply:
It's just a best practice, never use more than one line in the log. In the other case, you lose the opportunity to parse your logs as a whole thing